Project/Project/src/app/guild/guild.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { RealmsService } from '../realms.service';
+import { Realms } from '../realms';
+import { GuildService } from '../guild.service';
+import { Guild } from '../guild';
+import { Router, ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-guild',
+  templateUrl: './guild.component.html',
+  styleUrls: ['./guild.component.scss']
+})
+export class GuildComponent implements OnInit {
+  guild: string;
+  realm: string;
+  realms: Realms[] = [];
+  constructor(private realmsService: RealmsService, private guildService: GuildService, private router: Router) { }
+
+  ngOnInit() {
+    this.realmsService.getRealmslist().subscribe(realms => {
+      this.realms = realms;
```

Comment:
aca lo que te devuelve el subscribe podria tener otro nombre y luego asignarle al `this.realms = res.realms`. Para evitar lo del comentario de mas arriba

Reply:
Claro, ahora entiendo bien